src/centralserver/admin-service/core/src/test/java/org/niis/xroad/centralserver/restapi/service/CertificationServicesServiceTest.java

```diff
@@ -31,37 +31,65 @@
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.niis.xroad.centralserver.restapi.dto.CertificationService;
+import org.niis.xroad.centralserver.restapi.dto.CertificationServiceListItem;
+import org.niis.xroad.centralserver.restapi.dto.converter.ApprovedCaConverter;
 import org.niis.xroad.centralserver.restapi.entity.ApprovedCa;
 import org.niis.xroad.centralserver.restapi.repository.ApprovedCaRepository;
+import org.niis.xroad.centralserver.restapi.service.exception.NotFoundException;
 
 import java.util.List;
+import java.util.Optional;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
 class CertificationServicesServiceTest {
 
+    private static final Integer ID = 123;
+
     @Mock
     private ApprovedCaRepository approvedCaRepository;
+    @Mock
+    private ApprovedCaConverter approvedCaConverter;
```

Comment:
It would be great to use real implementations wherever we can, so converters that don't have any injectable dependencies are good candidates for that in unit tests.